tests/mocha/service.js

```diff
@@ -1,7 +1,54 @@
 import assert from 'assert'
 
-describe('my feature', () => {
-    it('should do stuff', () => {
+const sleep = (ms) => new Promise(resolve => setTimeout(resolve, ms))
+
+before(async () => {
+    await sleep(10)
```

Comment:
For what are we sleeping for?

Reply:
Nothing, just make sure async command works.
It can be replaced with browser.pause

Reply:
does it mean that they are executed at the same time?

Reply:
I think I missed something here.

I just want to make sure that mixing async and sync modes in same spec is not causing any unpredictable behavior. 
The reason for doing such test is that testFrameworkFnWrapper runs async function without fibers, nothing new here, it worked same way before, but still have to be tested.